dh_commitments/src/pedersen/mod.rs

```diff
@@ -0,0 +1,54 @@
+use algebra::groups::Group;
+use rand::Rng;
+use std::marker::PhantomData;
+
+use crate::{random_generators, DoublyHomomorphicCommitment, Error};
+
+use inner_products::{InnerProduct, MultiexponentiationInnerProduct};
+
+#[derive(Clone)]
+pub struct PedersenCommitment<G: Group> {
```

Comment:
You can change from `G: Group` to `C: ProjectiveCurve` if that helps you here (for efficiency or otherwise)

Reply:
:+1: 

I changed it to `ProjectiveGroup` in order to use the improved `VariableMSM` calculation of `MultiexponentiationInnerProduct`.